catalog/core/catalog-core-localstorageprovider/pom.xml

```diff
@@ -40,6 +40,11 @@
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-commons</artifactId>
         </dependency>
+        <dependency>
```

Comment:
There is a field I am using in the `ContentEncryptorDecryptor` that tells me how many extra bytes will be added to encrypted data. I thought about moving that field to the interface; but I don't feel it belongs in the interface since that value can change between implementations.

Reply:
❗️ Yeah. I still don't like this coupling. Suggest instead adding one more method to your service interface to `getExtraByteCount()`.